src/classes/Tools/User/UserSession.php

```diff
@@ -11,6 +14,9 @@ public function isLoggedIn()
 
     public function logout()
     {
+        $updateToken = new UpdateToken(new Database);
```

Comment:
Can we please use dependency injection in the constructor

Reply:
Done

Reply:
Wouldn't it be faster if loading of database be done for the function only as opposed to loading database for the whole class?

Reply:
in my opinion: its "bad practise" to instantiate classes this way it should be done through dependency injection :smile: